include/cellular_config_defaults.h

```diff
@@ -142,4 +142,13 @@
     #define CELLULAR_CONFIG_DEFAULT_RAT     ( 8 )  /* Set default RAT to CELLULAR_RAT_CATM1 @ref CellularRat_t. */
 #endif
 
+#ifndef LOOP_FOREVER
+  #ifdef TEST
+    extern int NumLoops;
+    #define LOOP_FOREVER() NumLoops--
+  #else
+    #define LOOP_FOREVER()1
```

Comment:
```suggestion
    #define LOOP_FOREVER()    1
```

Reply:
will remove this changes, keep only one configuration in Lab-Project-FreeRTOS-Cellular-Library\test\unit-test\cellular_config.h